python/trigger_ghebackup.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+import subprocess
+import sys
+
+
+def get_pid(name: str) -> list:
+    """Returns a list of pids, can be empty.
+    @attention: inspired by http://stackoverflow.com/a/26688998 on 09/02/2017
+    @author: Padraic Cunningham"""
+    if name:
+        return list(map(int, subprocess.check_output(["pidof", name]).split()))
```

Comment:
I would use a list comprehension here:

```python
return [int(pid) for pid in subprocess.check_output(["pidof", name]).split()]
```

since it makes it clearer from a cursory read that you are returning a list of integer PIDs